docs/src/examples/components/Popup/Types/PopupExample.shorthand.tsx

```diff
@@ -12,7 +12,7 @@ class PopupExample extends React.Component<any, any> {
     return (
       <Popup
         open={this.state.popupOpen}
-        onOpenChange={(e, newProps) => {
+        onChange={(e, newProps) => {
           alert(`Popup is requested to change its open state to "${newProps.open}".`)
```

Comment:
Exactly in this case, I would feel awkward to introduce onOpen and onClose for setting the value of the state to true and false, rather then having the onChange for handling both cases.